src/common/marlin_server.cpp

```diff
@@ -421,7 +421,7 @@ void marlin_server_print_start(const char *filename) {
     if ((marlin_server.print_state == mpsIdle) || (marlin_server.print_state == mpsFinished) || (marlin_server.print_state == mpsAborted)) {
         media_print_start(filename);
         for (int id = 0; id < MARLIN_MAX_CLIENTS; id++)
-            marlin_server.client_changes[id] |= MARLIN_VAR_MSK(MARLIN_VAR_FILEPATH);
+            marlin_server.client_changes[id] |= MARLIN_VAR_MSK(MARLIN_VAR_FILENAME);
```

Comment:
Shouldn't you notify both? LFN filename and SFN filepath?

Reply:
commit `f7b231a`